pkg/config/vpn-server.go

```diff
@@ -5,21 +5,25 @@
 package config
 
 import (
+	"fmt"
+
 	"github.com/caarlos0/env/v10"
-	"github.com/gardener/vpn2/pkg/network"
 	"github.com/go-logr/logr"
+
+	"github.com/gardener/vpn2/pkg/network"
 )
 
 type VPNServer struct {
-	ServiceNetworks []network.CIDR `env:"SERVICE_NETWORKS" envDefault:"100.64.0.0/13"`
-	PodNetworks     []network.CIDR `env:"POD_NETWORKS" envDefault:"100.96.0.0/11"`
-	NodeNetworks    []network.CIDR `env:"NODE_NETWORKS"`
-	VPNNetwork      network.CIDR   `env:"VPN_NETWORK"`
-	PodName         string         `env:"POD_NAME"`
-	StatusPath      string         `env:"OPENVPN_STATUS_PATH"`
-	IsHA            bool           `env:"IS_HA"`
-	HAVPNClients    int            `env:"HA_VPN_CLIENTS"`
-	LocalNodeIP     string         `env:"LOCAL_NODE_IP" envDefault:"255.255.255.255"`
+	ServiceNetworks  []network.CIDR `env:"SERVICE_NETWORKS" envDefault:"100.64.0.0/13"`
+	PodNetworks      []network.CIDR `env:"POD_NETWORKS" envDefault:"100.96.0.0/11"`
+	NodeNetworks     []network.CIDR `env:"NODE_NETWORKS"`
```

Comment:
Same questions as above with regards to the slice handling of the `env` package.